pyproject.toml

```
@@ -12,6 +12,10 @@ homepage =  "https://github.com/nat-n/poethepoet"
 python = "^3.6"
 pastel = "^0.2.0"
 tomli = "^1.0.0"
+poetry = {version = "^1.1.7", allow-prereleases = true, optional = true}
```

```suggestion
poetry = {version = "^1.0", allow-prereleases = true, optional = true}
```
The example from the docs suggests using `"^1.0"` is there any reason to be more prescriptive?